src/Lazy_Load_Script.php

```diff
@@ -77,7 +77,7 @@ public function print_script() {
 			script = document.createElement( 'script' );
 			script.id = 'native-lazyload-fallback';
 			script.type = 'text/javascript';
-			script.src = '<?php echo esc_js( $this->get_fallback_script_url() ); ?>';
+			script.src = '<?php echo wp_json_encode( esc_url(  $this->get_fallback_script_url() ) ); ?>';
```

Comment:
```suggestion
			script.src = '<?php echo wp_json_encode( esc_url( $this->get_fallback_script_url() ) ); ?>';
```

Does this still work with `wp_json_encode`? Did you test it?

Reply:
Yes, functionality is not affected.